cg/apps/demultiplex/sample_sheet/validate.py

```diff
@@ -10,8 +10,10 @@
     SampleBcl2Fastq,
     SampleDragen,
```

Comment:
```suggestion
    SampleBclConvert,
```

I think this might be another instance where dragen gets used but should be a more specific software

Reply:
Agreed! Let's  fix in a future PR